src/shared/pages/team/teamSearch/teamSearch.tsx

```diff
@@ -43,14 +57,27 @@ const TeamSearch: React.FC<TeamSearchProps> = ({ history }) => {
             <h1 className="govuk-heading-xl">
                 Team search
             </h1>
-            <TypeAhead
-                clearable={true}
-                disabled={false}
-                getOptions={getTeamsForTypeahead}
-                label={'Teams'}
-                name={'Teams'}
-                onSelectedItemChange={setSelectedTeam}
-            />
+            <div className="govuk-grid-row">
+                {showDeactivated === 'false' && <div className="govuk-grid-column-three-quarters"><TypeAhead
```

Comment:
Minor: The typecast to Boolean would create a concise expression i.e. `{showDeactivated && <div....`

Reply:
`showDeactivated` would resolve to true when it should be falsed. 

I agree with the previous comment with changing the type-cast to boolean, then this change can be done.

Reply:
I've addressed this comment and changed it in a later update. Originally I had it that way as the Checkbox component uses a 'false' or 'true' string to map its state. I'm now converting that state to a boolean before storing it instead.